config/v1/types_infrastructure.go

```diff
@@ -101,6 +101,14 @@ type InfrastructureStatus struct {
 	// +kubebuilder:default=HighlyAvailable
 	// +kubebuilder:validation:Enum=HighlyAvailable;SingleReplica
 	InfrastructureTopology TopologyMode `json:"infrastructureTopology"`
+
+	// cpuPartitioning expresses that CPU partitioning is a currently enabled feature in the cluster.
+	// CPU Partitioning means that this cluster can support partitioning workloads to specific CPU Sets
```

Comment:
This documentation should note the valid values and behaviour when omitted
// Valid values are None and AllNodes. When omitted, the default value is None.

Reply:
Updated, should be all set.